### DISCUSSION TITLE
Should we increase the bounty amounts?

### DISCUSSION THREAD

**Discussion by rickstaa** [livepeer/bounties#43] [Polls]
Reposting this here so we can do a poll.

> Just a follow up conversation around the LPT reward price given to people who complete a bounty.
> 
> I think it is imperative that we offer relatively high rewards for people completing bounties.
> 
> A number of benefits occur:
> 1. People are motivated to try their first bounty and to start digging into the code, especially if they are new to Livepeer
> 2. It builds enthusiasm around Livepeer and introduces a curiosity of "maybe I could get involved full/part time" since the rewards here are so lucrative.
> 3. It shortens the time frame for how quickly a bounty will get completed by creating an opportunity for competition.
> 
> When considering an LPT amount, I think we could offer 1.5X to 2X the current amounts.
> 
> And in the future, with the success of the program, we could possibly request a separate amount from the Treasury dedicated to Bounties.

_Originally posted by @Titan-Node in https://github.com/livepeer/bounties/discussions/42_

**Comment by rickstaa** [comment]
Hey @Titan-Node,

Thanks for your feedback. I agree that offering more generous rewards for these bounties makes sense, as it could attract high-quality Open Source contributors who might even make this a full-time profession. To achieve this, we need to consider the lack of security these contributors have compared to working at a company and the volatility of payments in LPT.

Currently, since the money for the AI Bounties (5000 LPT) was allocated by the community after a treasury vote, I aimed for what I believe is fair compensation. This was based on my estimation of the time a mid-level developer would need for the task, adding a 40% safety margin. I then looked at the average rate for a software engineering consultant in the US and converted that amount to the one-month average price of Livepeer (see [this Discord post](https://discord.com/channels/423160867534929930/1187806216185974934/1269239098418266142)).

However, if the community is open to adding a bonus amount, such as an extra 50%, I'm happy to factor that into the calculation 👍🏻.

I'm curious to hear the thoughts on this from the rest of the community.

**Comment by JJassonn69** [comment]
As a beginner bounty hunter, I see these rewards as way to increase incentive for people to get involved in the ecosystem. Increasing the reward by too much might have the opposite effect and just attract people who are in it for the money and as a one time thing rather than someone who will be a part of the community. We have already seen that there isn't a lack of bounty hunters with our previous test. 
Thus I vote to increase by **1.5x**.

**Comment by Tibia2000** [comment]
> There appears to be no shortage of people wanting to contribute to the Livepeer ecosystem. The main limitation has been the speed of the AI team in posting enough bounties to meet the interest. We plan to increase our pace in the coming months ⚡

If there is no shortage of bounty hunters, it indicates they are adequately valued. The primary goal of funding is to achieve maximum development at minimal cost, similar to any company. If bounties are there but there are no individuals willing to complete the tasks, then the bounties should be increased to encourage ppl to execute them. That is how the fair value is found, by matching price ask with price bid. 

However, a downside of increasing bounties excessively is that it might encourage people to expect payment for any of their contributions. While it's fair for people to receive compensation for their work, this could potentially discourage current open-source developers who have contributed significantly in the past without compensation out of passion for the Livepeer project. It might be more effective to fund a day to day contractor dev for Livepeer INC than increasing bounties for such a small tasks.
